tool/planet/start.go

```diff
@@ -525,6 +532,25 @@ func setupEtcd(config *Config) error {
 	return nil
 }
 
+// setupISCSI runs ISCSI related setup tasks.
+// If OpenEBS is disabled in the config then
+// we disable(mask) the ISCSI related services.
+func setupISCSI(config *Config) error {
+	if !config.OpenEBSEnabled {
```

Comment:
Looks like the function is a no-op if `config.OpenEBSEnabled == true` - why not move the condition check outside and pass it just the rootfs path?

Reply:
setupISCSI contains all steps that are needed in order to setup ISCSI. The check !config.OpenEBSEnabled is closely related to the setup process. As such it would be better to leave it as is. Let it be encapsulated as part of the setupISCSI collection of steps.